client/extensions/woocommerce/app/reviews/review-reply.js

```diff
@@ -140,7 +140,6 @@ class ReviewReply extends Component {
 					className="reviews__reply-content"
 					dangerouslySetInnerHTML={ { __html: content } } //eslint-disable-line react/no-danger
 					// Sets the rendered comment HTML correctly for display.
-					// Also used for comments in `comment-detail/comment-detail-comment.jsx`
```

Comment:
See: https://github.com/Automattic/wp-calypso/pull/20738/files#r156430281

Reply:
Updated in 0a613e41058b64b8827af927f5ee7c2a68dabcbd.